pkg/eks/eks.go

```diff
@@ -378,47 +376,30 @@ func (c *ClusterProvider) doListClusters(chunkSize int64, printer printers.Outpu
 }
 
 // GetCluster display details of an EKS cluster in your account
-func (c *ClusterProvider) GetCluster(clusterName string, output printers.Type) error {
-	printer, err := printers.NewPrinter(output)
-	if err != nil {
-		return err
-	}
-
-	if output == "table" {
-		addSummaryTableColumns(printer.(*printers.TablePrinter))
-	}
-
-	return c.doGetCluster(clusterName, printer)
-}
-
-func (c *ClusterProvider) doGetCluster(clusterName string, printer printers.OutputPrinter) error {
+func (c *ClusterProvider) GetCluster(clusterName string) (*awseks.Cluster, error) {
 	input := &awseks.DescribeClusterInput{
 		Name: &clusterName,
 	}
+
 	output, err := c.Provider.EKS().DescribeCluster(input)
 	if err != nil {
-		return errors.Wrapf(err, "unable to describe control plane %q", clusterName)
+		return &awseks.Cluster{}, errors.Wrapf(err, "unable to describe control plane %q", clusterName)
```

Comment:
The AWS API likes to return pointers so this is probably left over from switching but I think either return `nil` for the error case or return a value (which I'm partial to) and `awseks.Cluster{}` in the error case, wdyt?

Reply:
sounds good to me!